challenges/back-end-challenge/flask_app.py

```diff
@@ -0,0 +1,52 @@
+
+# A very simple Flask Hello World app for you to get started with...
+
+from flask import Flask, request, Response
+import json
+from os import path
+
+app = Flask(__name__)
+
+sorted_data = []
```

Comment:
I noticed this variable is not used, what is this here for?

Reply:
I forgot to remove this unused variable. It has no use in the API,